edb/schema/expr.py

```diff
@@ -213,44 +258,48 @@ def compare_values(cls, ours, theirs, *,
         return basecoef + (1 - basecoef) * compcoef
 
 
-def imprint_expr_context(qltree, modaliases):
+def imprint_expr_context(
+    qltree: qlast_.Base,
+    modaliases: Mapping[Optional[str], str],
+) -> qlast_.Base:
     # Imprint current module aliases as explicit
     # alias declarations in the expression.
 
-    if (isinstance(qltree, qlast.BaseConstant)
+    if (isinstance(qltree, qlast_.BaseConstant)
             or qltree is None
-            or (isinstance(qltree, qlast.Set)
+            or (isinstance(qltree, qlast_.Set)
                 and not qltree.elements)
-            or (isinstance(qltree, qlast.Array)
-                and all(isinstance(el, qlast.BaseConstant)
+            or (isinstance(qltree, qlast_.Array)
+                and all(isinstance(el, qlast_.BaseConstant)
                         for el in qltree.elements))):
         # Leave constants alone.
         return qltree
 
-    if not isinstance(qltree, qlast.Command):
-        qltree = qlast.SelectQuery(result=qltree, implicit=True)
+    if not isinstance(qltree, qlast_.Command):
+        qltree = qlast_.SelectQuery(result=qltree, implicit=True)
     else:
         qltree = copy.copy(qltree)
         qltree.aliases = list(qltree.aliases)
 
-    existing_aliases = {}
+    existing_aliases: Dict[str, str] = {}
     for alias in qltree.aliases:
-        if isinstance(alias, qlast.ModuleAliasDecl):
+        if isinstance(alias, qlast_.ModuleAliasDecl):
             existing_aliases[alias.alias] = alias.module
 
     aliases_to_add = set(modaliases) - set(existing_aliases)
-    for alias in aliases_to_add:
+    for alias_name in aliases_to_add:
         qltree.aliases.append(
-            qlast.ModuleAliasDecl(
-                alias=alias,
-                module=modaliases[alias],
+            qlast_.ModuleAliasDecl(
+                alias=alias_name,
+                module=modaliases[alias_name],
             )
         )
 
     return qltree
 
 
-def get_expr_referrers(schema, obj):
+def get_expr_referrers(schema: s_schema.Schema, 
+                       obj: so.Object) -> Dict[Any, Any]:
```

Comment:
The correct annotation for the return type is `Dict[so.Object, str]`

Reply:
You answered while I was asking the question, so fast! I do it.